app/src/main/java/com/github/sdpteam15/polyevents/fragments/MapsFragment.kt

```diff
@@ -40,6 +44,20 @@ class MapsFragment : Fragment(), OnMapReadyCallback, OnPolylineClickListener,
         val saveNewAreaButton:View = view.findViewById(R.id.acceptNewArea)
         addNewAreaButton.setOnClickListener(View.OnClickListener { view -> GoogleMapHelper.createNewArea() })
         saveNewAreaButton.setOnClickListener(View.OnClickListener { view -> GoogleMapHelper.saveNewArea() })
+
+        locationButton = view.findViewById(R.id.id_location_button)
+        locationButton.setOnClickListener {
+            switchLocationOnOff()
+        }
+        locationButton.tag = R.drawable.ic_location_on
+
+        val locateMeButton = view.findViewById<FloatingActionButton>(R.id.id_locate_me_button)
+        locateMeButton.setOnClickListener {
+            moveToMyLocation()
+        }
+
+        locateMeButton.tag = R.drawable.ic_locate_me
```

Comment:
What do you intend to use the tag for?

Reply:
I'm aware it's dirty and not best practices : it is for the test so that we can check if the resource drawable is changed when clicking on the button. When we will write more tests for this Google Map thing, it should be replaced by something cleaner :)